pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test: TestTenantLogic_zero failed

pkg/ccl/logictestccl/tests/3node-tenant/3node-tenant_test.TestTenantLogic_zero [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7574768?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7574768?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):

Fatal error:

```
panic: test timed out after 59m55s

```
Stack: 

```
goroutine 7178636 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantLogic_zero
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/47424cbdfa0dc4481b838b4a69ffa32f/logTestTenantLogic_zero2775791373
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantLogic_zero.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21577

--- a/pkg/cmd/teamcity-trigger/main.go
+++ b/pkg/cmd/teamcity-trigger/main.go
@@ -109,7 +109,8 @@ func runTC(queueBuild func(string, map[string]string)) {
 			} else {
 				opts["env.COCKROACH_KVNEMESIS_STEPS"] = "10000"
 			}
-		case baseImportPath + "sql/logictest", baseImportPath + "kv/kvserver":
+		case baseImportPath + "sql/logictest", baseImportPath + "kv/kvserver",
+			baseImportPath + "ccl/logictestccl":
 			// Stress heavy with reduced parallelism (to avoid overloading the
 			// machine, see https://github.com/cockroachdb/cockroach/pull/10966).
 			parallelism /= 2

